server/src/main/java/com/testsigma/service/TestsigmaLabDriverSettingsService.java

```diff
@@ -145,4 +147,21 @@ public void setWebCapabilities(TestDevice testDevice,
   public Integrations getLabDetails() throws IntegrationNotFoundException {
     return this.integrationsService.findByApplication(Integration.TestsigmaLab);
   }
+
+  private String getExecutionName() {
+    String name = "[Trial] - Mobile Inspection";
```

Comment:
@vikramvr20 can we OpenSource instead of Trial

Reply:
Addressed